### PR TITLE
feat: Merge EKS into main

### PR DESCRIPTION
**Pull request by alessio-creo**

### CODE REVIEW

File: src/main/60-eks.tf [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      {
+        Action = [
+          "ec2:Describe*",
+        ]
+        Effect   = "Allow"
+        Resource = "*"
+      },
+    ]
+  })
+}
```
Comment (micdes-pagopa) [inline src/main/60-eks.tf:16]: Do we need this? Check if the EKS module already assigns the correct role+policies

File: src/main/60-eks.tf [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "19.10.0"
+
+  cluster_name    = "${var.app_name}-eks-${var.env}"
+  cluster_version = "1.24"
```
Comment (micdes-pagopa) [inline src/main/60-eks.tf:23]: You can use the variable you have declared in the variables file

File: src/main/60-eks.tf [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+  cluster_addons = {
+    kube-proxy = {
+      addon_version = "v1.24.9-eksbuild.1"
+    }
+    vpc-cni = {
+      addon_version = "v1.12.2-eksbuild.1"
+    }
+    coredns = {
+      addon_version = "v1.9.3-eksbuild.2"
```
Comment (micdes-pagopa) [inline src/main/60-eks.tf:33]: Addon versions should be variables

File: src/main/60-eks.tf (hunk 1 above)
Comment (alessio-creo) [reply src/main/60-eks.tf:16]: Seems like the module create the IAM role (https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/modules/fargate-profile/main.tf)

File: src/main/60-eks.tf (hunk 2 above)
Comment (alessio-creo) [reply src/main/60-eks.tf:23]: Is meant to pin the version, but I'll set the variables

File: src/main/60-eks.tf (hunk 3 above)
Comment (alessio-creo) [reply src/main/60-eks.tf:33]: I'll set the variables

**Pull request merged**